src/components/screens/HistoryScreen/mappers/changeTrust.tsx

```diff
@@ -3,40 +3,31 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import BigNumber from "bignumber.js";
 import { AssetIcon } from "components/AssetIcon";
-import {
-  TransactionDetails,
-  TransactionType,
-  TransactionStatus,
-  HistoryItemData,
-} from "components/screens/HistoryScreen/types";
+import { HistoryItemUI } from "components/screens/HistoryScreen/types";
 import Icon from "components/sds/Icon";
 import { ThemeColors } from "hooks/useColors";
-import { t } from "i18next";
 import React from "react";
 
-interface ChangeTrustHistoryItemData {
-  operation: any;
-  stellarExpertUrl: string;
-  date: string;
-  fee: string;
-  themeColors: ThemeColors;
-}
+// interface ChangeTrustHistoryItemData {
+//   operation: any;
+//   stellarExpertUrl: string;
+//   date: string;
+//   fee: string;
+//   themeColors: ThemeColors;
+// }
 
 /**
  * Maps change trust operation data to history item data
  */
-export const mapChangeTrustHistoryItem = ({
-  operation,
-  stellarExpertUrl,
-  date,
-  fee,
-  themeColors,
-}: ChangeTrustHistoryItemData): HistoryItemData => {
+export const mapChangeTrustHistoryItem = (
```

Comment:
here you can see the simplification of the mappers: this function just needs to consume data from the shared lib and return the appropriate icons. That's it!